R/Task.R

```diff
@@ -127,6 +136,17 @@ makeTask = function(type, data, weights = NULL, blocking = NULL, fixup.data = "w
     }
   }
 
+  if (spatial == TRUE) {
+    # check if coords are named 'x' and 'y'
+    if (!any(colnames(data) == "x" | colnames(data) == "y")) {
+      stop("Please rename coordinates in data to 'x' and 'y'.")
+    }
+  }
+  # check if setting 'spatial = TRUE' was forgotten by accident
+  if (spatial == FALSE && any(colnames(data) == "x" & any(colnames(data) == "y"))) {
```

Comment:
Your parentheses seem off here -- please add a test for this code and the check above.

Reply:
https://github.com/mlr-org/mlr/pull/1891/commits/4728a2a16183bdebbe0375c774a957c1194e9335 and https://github.com/mlr-org/mlr/pull/1891/commits/550db62ded6c9020f3fe862b30650f2b4996e373